app/assets/stylesheets/application.scss

```diff
@@ -82,3 +82,7 @@ body {
 img {
   max-width: 100%;
 }
+
+.title {
+  font-size: 36px;
```

Comment:
Add top padding of 20px to keep more space above the titles.

Reply:
yeah, make sense! Added padding.